### PR TITLE
feat/health-check-endpoint

### PR DESCRIPTION
**Pull request by bernardinorafael**
This PR adds improvements to health check endpoint.

### CODE REVIEW

**Comment by bernardinorafael** [comment]
resolves detrash/recy-network#164

File: src/modules/health/health.controller.ts [hunk 1]
```diff
@@ -1,13 +1,70 @@
...
+    this.redis = new Redis({
+      host: process.env.REDIS_HOST,
+      port: parseInt(process.env.REDIS_PORT ?? '6379'),
+      retryStrategy: (times) => {
+        if (times >= 3) return null;
+        return 1000 * 10; // 10 sec
+      },
+    });
+
+    this.redis.on('error', (err) => console.error(err));
```
Comment (yurimutti) [inline src/modules/health/health.controller.ts]: Could you please create an issue for creating a Redis service?

**Review by yurimutti — DISMISSED** [review]
LGFM

File: src/modules/health/health.controller.ts (hunk 1 above)
Comment (yurimutti) [reply src/modules/health/health.controller.ts]: and please create a Todo comment to change console.error to pino logger.

**Pull request merged**